rocks-lib/src/lua_rockspec/serde_util.rs

```diff
@@ -68,3 +70,66 @@ where
         })
         .try_collect()
 }
+
+pub(crate) enum DisplayLuaValue {
+    // NOTE(vhyrro): these are not used in the current implementation
+    // Nil,
+    // Number(f64),
+    Boolean(bool),
+    String(String),
+    List(Vec<Self>),
+    Table(Vec<DisplayLuaKV>),
+}
+
+pub(crate) struct DisplayLuaKV {
+    pub(crate) key: String,
+    pub(crate) value: DisplayLuaValue,
+}
+
+impl Display for DisplayLuaValue {
```

Comment:
suggestion: some tests would be nice.

Reply:
Added a test for the DisplayLuaValue to make sure it's all working nicely, I think it should suffice, but do let me know if I should extend it some way.